kairos-cli/bin/crypto/public_key.rs

```diff
@@ -0,0 +1,29 @@
+use crate::crypto::error::CryptoError;
+use casper_types::bytesrepr::FromBytes;
+use casper_types::bytesrepr::ToBytes;
+
+#[derive(Clone)]
+pub struct CasperPublicKey(pub casper_types::PublicKey);
+
+impl CasperPublicKey {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, CryptoError> {
+        let (public_key, _remainder) =
+            casper_types::PublicKey::from_bytes(bytes).map_err(|_e| {
+                CryptoError::Serialization {
+                    context: "public key",
+                }
+            })?;
+        Ok(Self(public_key))
+    }
+
+    pub fn from_key(public_key: casper_types::PublicKey) -> Self {
```

Comment:
I don't think there is a benefit having this function, users would simply pattern-match to get the inner value

Reply:
Removed in https://github.com/cspr-rad/kairos/pull/17/commits/72d333c22d8de19c00ea4debfa6d9a3a808fb893.